core/storage/suggestion/gae_models_test.py

```diff
@@ -1707,3 +1823,53 @@ def test_export_data_nontrivial(self) -> None:
             .export_data(self.CONTRIBUTOR_USER_ID))
 
         self.assertEqual(expected_data, user_data)
+
+    def test_get_model_association_to_user(self) -> None:
+        translation_model = suggestion_models.TranslationContributionStatsModel
+        self.assertEqual(
+            base_models.MODEL_ASSOCIATION_TO_USER.MULTIPLE_INSTANCES_PER_USER,
+            translation_model.get_model_association_to_user()
+        )
+
+    def test_get_all_by_user_id(self) -> None:
+        translation_model = suggestion_models.TranslationContributionStatsModel
+        self.assertEqual(
+            translation_model.get_all_by_user_id(
+                'author_1'
+            ),
+            []
+        )
+
+    def test_get_export_policy(self) -> None:
+        dictionary = {
+            'language_code':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'contributor_user_id':
+                base_models.EXPORT_POLICY.NOT_APPLICABLE,
+            'topic_id':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'submitted_translations_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'submitted_translation_word_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'accepted_translations_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'accepted_translations_without_reviewer_edits_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'accepted_translation_word_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'rejected_translations_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'rejected_translation_word_count':
+                base_models.EXPORT_POLICY.EXPORTED,
+            'contribution_dates':
+                base_models.EXPORT_POLICY.EXPORTED,
+                'created_on': base_models.EXPORT_POLICY.NOT_APPLICABLE,
```

Comment:
Something's off with your indentation here.

I'm not really sure why you're breaking after the colon for most of these, too.

Reply:
Resolved this issue.